config/chime_science_run_gpu.yaml

```diff
@@ -60,6 +61,7 @@ downsample_freq: 8
 factor_upchan: 128
 factor_upchan_out: 16
 num_frb_total_beams: 1024
+num_sub_freqs: 128
```

Comment:
This variable is the same as `factor_upchan` in the FRB options.   We should merge this down to one variable.